pypreprocess/tests/test_reporting.py

```diff
@@ -0,0 +1,27 @@
+import os
+from nilearn.datasets import fetch_haxby
+from pypreprocess.nipype_preproc_spm_utils import (do_subjects_preproc,
+                                                    SubjectData)
+
+def test_reporting():
+    # fetch HAXBY dataset
+    haxby_data = fetch_haxby(subjects=1)
+
+    # set output dir
+    output_dir = os.path.join(os.path.dirname(haxby_data.mask),"haxby_runs")
```

Comment:
```suggestion
    output_dir = os.path.join(os.path.dirname(haxby_data.mask), "haxby_runs")
```

Reply:
We could do the same thing done in circleci but SPM+Matlab installation takes forever. Maybe a pure python example instead of the SPM one?

Reply:
Agreed. Is it possible to call pure Python preprocessing instead ?

Reply:
It is just that the pure python preprocessing module does not use the new reporting yet. I only made changes in the spm module in the previous PR. But I could do that now. Should make those changes in this PR or would you rather prefer a separate PR for that?